src/webxdc/integration.rs

```diff
@@ -57,14 +57,34 @@ impl Context {
     }
 
     // Check if a Webxdc shall be used as an integration and remember that.
-    pub(crate) async fn update_webxdc_integration_database(&self, msg: &Message) -> Result<()> {
-        if msg.viewtype == Viewtype::Webxdc && msg.param.get_int(Param::WebxdcIntegration).is_some()
-        {
-            self.set_config_internal(
-                Config::WebxdcIntegration,
-                Some(&msg.id.to_u32().to_string()),
-            )
-            .await?;
+    pub(crate) async fn update_webxdc_integration_database(
+        &self,
+        msg: &mut Message,
+        context: &Context,
+    ) -> Result<()> {
+        if msg.viewtype == Viewtype::Webxdc {
+            let is_integration = if msg.param.get_int(Param::WebxdcIntegration).is_some() {
+                true
+            } else if msg.chat_id.is_self_talk(context).await? {
```

Comment:
Is there a way to send such a webxdc file to the self-chat w/o triggering any "integration" side effects? I think there should be a way, otherwise this breaks a normal usage of the self-chat when you just want to save smth or share with another device

Reply:
Btw, since 42a7cd3eea38a6ee04a9463c81df2f41157db4bc core supports creation of self-only groups, i.e. w/o other recipients, maybe such a separate group should be better used for replacing webxdc integrations

Reply:
well, you do not get these files "just so" currently and integration is only triggered locally.  they are created by devs/power-users. this is okay in the current experimental stage. 

by deletion of the file, integration is cleared, so there is an easy way out (i just added a commit to show that in the summary)

i think, if one created such a file, in most one wants to try it out at the current stage :) when going out of experimental and keep this path, we probably add UI to ask for integration (i can imagine an "Integrate" instead of a "Start" button, showing a dialog explaining what will go on). but we're not there.

the idea of this PR is to get things in without UI effort, to get experiences.

Reply:
Anyway, using a separate chat is safer, or if we want to use the self-chat, this should be enabled by a Config option (this doesn't need a UI on Desktop as you can enable anything via env, but may be complicated on other platforms). Otherwise it may be unexpected for a regular user that forwarding smth to the self-chat replaces e.g. the standard maps

Reply:
there is no "standard map" currently, the "experimental map" is enabled by a config option only. 

the case that one 
- enables "experimental maps"
- adaps the map oneself (or gets it from someone else)
- sends it to saved messages

... and then wonders why the map is replaces seems pretty constructed :) as said, when we go out of experimental, things will change.

perspectively, when we device to allow extensions beyond experimental, the "Saved Messages" also has benefits as it is already there. 

but for now, let's just try the current approach and not put more effort in that at the current stage that may get removed alltogether.